lib/mocha.js

```diff
@@ -277,15 +283,67 @@ Mocha.prototype.invert = function() {
  * @param {Boolean} ignore
  * @return {Mocha}
  * @api public
- * @param {boolean} ignore
- * @return {Mocha}
  */
```

Comment:
Some changes like this would be better just applied to master before this PR than applied during this PR; then if we ended up having to revert we wouldn't have to cherry-pick the unrelated changes to keep them, plus it would reduce clutter when reviewing the PR. (I figure something similar goes for `"` to `'` conversion in bin/_mocha, by the way.)